hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/GcsEventsSource.java

```diff
@@ -201,13 +202,14 @@ private MessageBatch processMessages(List<ReceivedMessage> receivedMessages) {
 
       MessageValidity messageValidity = message.shouldBeProcessed();
       if (messageValidity.getDecision() == DO_SKIP) {
-        LOG.info("Skipping message: " + messageValidity.getDescription());
+        LOG.debug("Skipping message: " + messageValidity.getDescription());
+        skippedMsgCount++;
         continue;
       }
 
       messages.add(msgStr);
     }
-
+    LOG.info("Messages received: " + receivedMessages.size() + ", toBeProcessed: " + messages.size() + ", skipped: " + skippedMsgCount);
```

Comment:
Please use placeholders (`{}`) instead of concatenating strings.

Reply:
Updated